LongestPalindrome.java

```diff
@@ -0,0 +1,22 @@
+import java.util.*;
+
+public class LongestPalindrome {
+    // TC: O(N), where N is the number of chars in String s
+    // SC: O(N), where N is the number of unique chars in String s
```

Comment:
Unique chars at most can be 26 , all lower case characters which is constant. So space will be O(1)